src/components/message/styles.scss

```diff
@@ -194,15 +195,16 @@
   }
 
   &__author-name {
-    font-weight: 500;
+    display: var(--message-author-name-display);
+    font-weight: 700;
     font-size: 12px;
     line-height: 15px;
     color: theme.$color-greyscale-12;
   }
 
   &.messages__message--last-in-group {
     .message__author-avatar {
-      display: block;
+      visibility: visible;
```

Comment:
I did this change because the css was breaking in case of using "display: none" (for images)

<img width="590" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/78154205-9987-42f0-b8fe-ed4fbca427ec">


After
<img width="595" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/d0408806-6baf-4434-bdaf-4c77af169244">

Reply:
ohhhh @dalefukami this is actually an issue on dev right now, i will pluck this out in a separate PR and merge it.